src/configuration-wire/configuration-wire-helper.ts

```diff
@@ -0,0 +1,75 @@
+import ApiEndpoints from '../api-endpoints';
+import FetchHttpClient, {
+  IBanditParametersResponse,
+  IHttpClient,
+  IUniversalFlagConfigResponse,
+} from '../http-client';
+
+import { ConfigurationWireV1, IConfigurationWire } from './configuration-wire-types';
+
+export type SdkOptions = {
+  sdkName: string;
+  sdkVersion: string;
+  baseUrl?: string;
+};
+
+/**
+ * Helper class for fetching and converting configuration from the Eppo API(s).
+ */
+export class ConfigurationWireHelper {
+  private httpClient: IHttpClient;
+
+  /**
+   * Build a new ConfigurationHelper for the target SDK Key.
+   * @param sdkKey
+   */
+  public static build(
+    sdkKey: string,
+    opts: SdkOptions = { sdkName: 'android', sdkVersion: '4.0.0' },
```

Comment:
is there any reason we're choosing these values for defaults? They seem extremely off from being default